<?xml version="1.0" ?>
<launch>

  <arg name="config" default="$(find topological_navigation)/config/sample_edge_reconfig_groups.yaml"/>
  <rosparam command="load" file="$(arg config)"/>

  <node pkg="topological_navigation" type="reconf_at_edges_server.py" name="reconf_at_edges_server" output="screen" respawn="true">
    <remap from="/move_base/DWAPlannerROS/set_parameters" to="move_base/DWAPlannerROS/set_parameters" />
  </node>

</launch>
